docs/Installation.md

```diff
@@ -1,8 +1,134 @@
+
+export E2E_PAC_GITHUB_APP_PRIVATE_KEY=$(base64 < /PATH/TO/YOUR/DOWNLOADED/PRIVATE_KEY.pem)
+```
+
+<p>
+Navigate back to <a href="https://github.com/settings/apps">your GitHub App</a>, select Install App and select your GitHub org (the one that you're using in `MY_GITHUB_ORG` env var). Feel free to install it to all repositories of that organization or the forked repositories currently used by <a href="(https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go)">rhtap-demo</a> and <a href="(https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go">build tests</a>
+</p>
+
+</details>
+
+For bootstrapping a cluster, run the following command:
+
+   ```bash
+      make local/cluster/prepare
+   ```
+
+More information about how to deploy RHTAP
+are in the [infra-deployments](https://github.com/redhat-appstudio/infra-deployments) repository.
+
+### Building and running the e2e tests
+
+Most of the tests could require you to have specific container image repo's created (if you're using your own container image org/user account (`QUAY_E2E_ORGANIZATION`) or your own GitHub organization (`MY_GITHUB_ORG`)
+In that case, before you run the test, make sure you have created
+* `test-images` repo in quay.io, i.e. `quay.io/<QUAY_E2E_ORGANIZATION>/test-images` and make it **public**
+  * also make sure that the docker config, that is encoded in the value of `QUAY_TOKEN` environment variable, contains a correct credentials required to push to `test-images` repo. And make sure the robot account or user account has the **write** permissions set for `test-images` repo which is required by the tests to push the generated artifacts.
+* fork following GitHub repositories to your org (specified in `MY_GITHUB_ORG` env var)
+  * https://github.com/redhat-appstudio-qe/devfile-sample-hello-world (for running build-service tests)
+  * https://github.com/redhat-appstudio-qe/hacbs-test-project (for rhtap-demo test)
+  * https://github.com/redhat-appstudio-qe/strategy-configs (for rhtap-demo test)
+
+You can use the following make target to build and run the tests:
+   ```bash
+      make local/test/e2e
+   ```
+
+Or build and run the tests without scripts:
+1. Install dependencies and build the tests:
+
+   ``` bash
+   # Install dependencies
+   $ go mod tidy
+   # Copy the dependencies to vendor folder
+   $ go mod vendor
+   # Create `e2e-appstudio` binary in bin folder. Please add the binary to the path or just execute `./bin/e2e-appstudio`
+   $ make build
+   ```
+
+2. Run the e2e tests:
+The `e2e-appstudio` command is the root command that executes all test functionality. To obtain all available flags for the binary please use `--help` flags. All ginkgo flags and go tests are available in `e2e-appstudio` binary.
+
+
+   ```bash
+      `./bin/e2e-appstudio`
+   ```
+
+The instructions for every test suite can be found in the [tests folder](tests), e.g. [has Readme.md](tests/has/README.md).
```

Comment:
`tests/has/README.md` does not exists now, would it be good to add an example for readme which is present like rhtap-demo or byoc?